src/components/externalLinks/ExternalLinks.component.tsx

```diff
@@ -123,6 +128,61 @@ export const NoMatchingResults = (): JSX.Element => {
     return <GenericFilterEmptyState />
 }
 
+const ExternalLinkIframeModal = ({ selectedExternalLink, handleCloseModal }) => {
+    const handleOpenExternalLink = () => {
+        window.open(selectedExternalLink.externalLinkURL, '_blank')
+    }
+
+    return (
+        <VisibleModal2 close={handleCloseModal}>
+            <div
+                className="flexbox-col dc__position-abs br-8 dc__top-12 dc__bottom-12 dc__right-12 dc__left-12 bcn-0"
+                onClick={stopPropagation}
+            >
+                <div className="flexbox dc__content-space px-20 py-12 dc__align-items-center dc__border-bottom">
+                    <div className="flexbox dc__gap-8 dc__align-items-center">
+                        <img
+                            className="icon-dim-20"
+                            src={selectedExternalLink.icon}
+                            alt={selectedExternalLink.label}
+                            onError={onImageLoadError}
+                        />
+                        <span className="cn-9 fs-16 fw-6 lh-24">{selectedExternalLink.label}</span>
+                    </div>
+                    <div className="flexbox dc__gap-8">
+                        <Button
+                            ariaLabel="external-link-open"
+                            dataTestId="external-link-open"
+                            icon={<ICArrowOut className="scn-6" />}
+                            variant={ButtonVariantType.borderLess}
+                            size={ComponentSizeType.xs}
+                            component={ButtonComponentType.button}
+                            style={ButtonStyleType.neutral}
+                            onClick={handleOpenExternalLink}
+                            showAriaLabelInTippy={false}
+                        />
+                        <Button
+                            ariaLabel="external-link-modal-close"
+                            dataTestId="external-link-modal-close"
+                            icon={<ICClose />}
+                            variant={ButtonVariantType.borderLess}
+                            size={ComponentSizeType.xs}
+                            component={ButtonComponentType.button}
+                            style={ButtonStyleType.negativeGrey}
+                            onClick={handleCloseModal}
+                            showAriaLabelInTippy={false}
+                        />
+                    </div>
+                </div>
+                <iframe
+                    className="flex-grow-1 dc__no-border dc__bottom-radius-8"
```

Comment:
Iframe must have unique title, and check with @abhibhaw if we can restrict permissions through tags like sandbox, refer: IframeElement in fe-common-lib.